packages/fx-core/tests/plugins/resource/aad/unit/generateArmTemplates.test.ts

```diff
@@ -2,176 +2,58 @@
-    // const mockedSolutionDataContext = {
-    //   Plugins: activeResourcePlugins,
-    //   PluginOutput: {
-    //     "fx-resource-bot": {
-    //       outputs: {
-    //         domain: "test_bot_domain_url",
-    //       },
-    //     },
-    //     "fx-resource-aad-app-for-teams": {},
-    //   },
-    // };
-    // chai.assert.isTrue(result.isOk());
-    // if (result.isOk()) {
-    //   const expectedResult = mockSolutionUpdateArmTemplates(
-    //     mockedSolutionDataContext,
-    //     result.value
-    //   );
-
-    //   const expectedVariablesFilePath = path.join(
-    //     __dirname,
-    //     "expectedBicepFiles",
-    //     "bot_variables.bicep"
-    //   );
-    //   chai.assert.strictEqual(
-    //     expectedResult.Orchestration.VariableTemplate!.Content,
-    //     fs.readFileSync(expectedVariablesFilePath, ConstantString.UTF8Encoding)
-    //   );
-    // }
-  });
-
-  it("generate arm templates: tab and bot", async function () {
-    // Act
-    return;
-    // const activeResourcePlugins = [
-    //   ResourcePlugins.Bot,
-    //   ResourcePlugins.Aad,
-    //   ResourcePlugins.FrontendHosting,
-    //   ResourcePlugins.SimpleAuth,
-    // ];
-    // const pluginContext: PluginContext = await TestHelper.pluginContext(new Map(), true, false);
-    // pluginContext.projectSettings!.solutionSettings = {
-    //   name: "test_solution",
-    //   version: "1.0.0",
-    //   activeResourcePlugins: activeResourcePlugins,
-    // } as AzureSolutionSettings;
-    // const result = await AADPlugin.generateArmTemplates(pluginContext);
-
-    // // Assert
-    // const mockedSolutionDataContext = {
-    //   Plugins: activeResourcePlugins,
-    //   PluginOutput: {
-    //     "fx-resource-bot": {
-    //       outputs: {
-    //         domain: "test_bot_domain_url",
-    //       },
-    //     },
-    //     "fx-resource-aad-app-for-teams": {},
-    //     "fx-resource-frontend-hosting": {
-    //       Outputs: {
-    //         domain: "test_frontend_hosting_domain_url",
-    //       },
-    //     },
-    //     "fx-resource-simple-auth": {},
-    //   },
-    // };
-    // chai.assert.isTrue(result.isOk());
-    // if (result.isOk()) {
-    //   const expectedResult = mockSolutionUpdateArmTemplates(
-    //     mockedSolutionDataContext,
-    //     result.value
-    //   );
-
-    //   const expectedVariablesFilePath = path.join(
-    //     __dirname,
-    //     "expectedBicepFiles",
-    //     "tabAndBot_variables.bicep"
-    //   );
-    //   chai.assert.strictEqual(
-    //     expectedResult.Orchestration.VariableTemplate!.Content,
-    //     fs.readFileSync(expectedVariablesFilePath, ConstantString.UTF8Encoding)
-    //   );
-    // }
+    const result = await aadPlugin.generateArmTemplates(pluginContext);
+
+    // Assert
+    chai.assert.isTrue(result.isOk());
+    console.log("result: ");
+    console.log(JSON.stringify(result));
+    if (result.isOk()) {
+      chai.assert.strictEqual(
```

Comment:
it's better to verify there has no other values return, such as modules or provision fields value.

Reply:
fixed.